openhcl/underhill_core/src/worker.rs

```diff
@@ -1165,12 +1165,6 @@ async fn new_underhill_vm(
         control_send,
     } = params;
 
-    if let Ok(kernel_boot_time) = std::env::var("KERNEL_BOOT_TIME") {
```

Comment:
Can we keep this so that we get the nice key-value tracing instead of string interpolation?

Reply:
I was thinking it wasn't worth the extra bit of unsafe code to pass this around as an env var when we can just log it immediately.

Reply:
This makes it easier to parse the logs in Kusto.

Reply:
Right, but it won't be unsafe with the new approach, right?

Reply:
Oh if we just do that, yeah duh.